tests/test_regression.py

```diff
@@ -19,10 +21,12 @@ def test_matching_mq_baseline(mq_test_df, theo_masses_df, mod_test, mq_baseline_
     )
     output_df = pd.read_csv(output_filepath, index_col=0)
 
-    pd.testing.assert_frame_equal(output_df, mq_baseline_df)
+    print(output_df.to_string(), file=regtest)
```

Comment:
Any idea why these statements fail on mismatched output? I'm sure they do, but it's hard to imagine how this has been done.